docs/content/how-to/disaster-recovery/backup-and-restore-oadp.md

```
@@ -905,31 +905,43 @@ Ensure your HostedCluster is configured with either:
   services:
   - service: APIServer
     servicePublishingStrategy:
-      type: LoadBalancer
-      loadBalancer:
-        hostname: api.basedomain.tld
+      type: Route
+      route:
+        hostname: api.example.com
 ```
 
+### Fixing OIDC After Restore
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)